reference-stores/next/pages/index.js

```diff
@@ -19,14 +19,16 @@ const Home = ({ page }) => {
 };
 
 export async function getStaticProps() {
-  const { pages } = await nacelleClient.query({
+  const { data } = await nacelleClient.query({
     query: CONTENT_PAGE_QUERY,
     variables: { handle: 'page-homepage' }
   });
+
   const { page } = await resolvePageData({
     client: nacelleClient,
-    page: pages.edges[0]?.node
+    page: data.pages.edges[0]?.node
```

Comment:
Might be a good opportunity to use optional chaining with `.at`:

```suggestion
    page: data.pages.edges.at(0)?.node
```